spec.html

```diff
@@ -1097,27 +1103,46 @@ <h1>DecorateClass ( elements, decorators )</h1>
 
     <emu-clause id=sec-decorate-element aoid=DecorateElement>
       <h1>DecorateElement ( element, keys )</h1>
-      <p>With parameters _element_, a Class Element and _keys_, a List of Property Keys and Private Names.</p>
+      <p>With parameters Class Element _element_ and _keys_:</p>
       <emu-alg>
         1. Let _extras_ be a new empty List.
         1. Let _finishers_ be a new empty List.
+        1. Let _placementKeys_ be *undefined*.
```

Comment:
I think that it should be `~empty~`, since `*undefined*` is the JavaScript value. Also, maybe it can just be used
```
1. If _element_.[[Placement]] is `"own"`, let _placementKeys_ to _keys_.[[OwnKeys]].
...
```
without declaring `_placementKeys_` first.

(example: https://tc39.github.io/ecma262/#sec-toprimitive)